dev-resources/profiles/cmi5/fixed.json

```diff
@@ -705,7 +705,7 @@
                 "en": "This pattern provides an option to use zero or more of the alernates for the Typical Session pattern."
             },
             "inScheme": "https://w3id.org/xapi/cmi5/v1.0",
-            "zeroOrMore": "https://w3id.org/xapi/cmi5#typicalsession"
+            "oneOrMore": "https://w3id.org/xapi/cmi5#typicalsession"
```

Comment:
does this track some change in the upstream CMI profile? just curious

Reply:
No, this change is 100% local. So this makes it _not_ the actual cmi5 profile, but instead a variant with a oneOrMore Pattern. None of the Profiles in `dev-resources` originally had a oneOrMore Pattern, hence why I had to make this change.

Reply:
The alternative would've been to make a custom Profile with all five different Patterns, but that seemed overkill.